src/framework/gracefulShutdown.ts

```diff
@@ -1,12 +1,35 @@
+import { logger } from "./logger.js";
 import { Application } from "../shared/types.js";
 import * as mongo from "./mongo.js";
+import { createTerminus, TerminusOptions } from "@godaddy/terminus";
+import { isDBHealthy } from "./crud-db.js";
 
-export const gracefulShutdown = (server: Application) => {
-  const atExitHandler = () => {
-    server.close();
-    mongo.disconnectDB();
-  };
+const loggerWrapper = (msg: string, err: Error) => {
+  if (err) {
+    logger.error(err.message);
+  } else {
+    logger.info(msg);
+  }
+};
```

Comment:
you should pass the message/error to the extra properties and write your own message, not sure what logs are written here